generators/addons/templates/azure-pipelines.yml

```diff
@@ -25,20 +25,27 @@ steps:
     verbose: false
 
 #start unit tests
-- task: Gulp@0
-  displayName: 'gulp test'
+- task: Npm@1
```

Comment:
@StfBauer these steps and the next two ones should probably be conditional to the fact that jest has been selected, what do you think?